notebooks/llm-chatbot/gradio_helper.py

```diff
@@ -0,0 +1,177 @@
+from typing import Callable, Literal
+import gradio as gr
+from uuid import uuid4
+
+
+chinese_examples = [
+    ["你好!"],
+    ["你是谁?"],
+    ["请介绍一下上海"],
+    ["请介绍一下英特尔公司"],
+    ["晚上睡不着怎么办？"],
+    ["给我讲一个年轻人奋斗创业最终取得成功的故事。"],
+    ["给这个故事起一个标题。"],
+]
+
+english_examples = [
+    ["Hello there! How are you doing?"],
+    ["What is OpenVINO?"],
+    ["Who are you?"],
+    ["Can you explain to me briefly what is Python programming language?"],
+    ["Explain the plot of Cinderella in a sentence."],
+    ["What are some common mistakes to avoid when writing code?"],
+    ["Write a 100-word blog post on “Benefits of Artificial Intelligence and OpenVINO“"],
+]
+
+japanese_examples = [
+    ["こんにちは！調子はどうですか?"],
+    ["OpenVINOとは何ですか?"],
+    ["あなたは誰ですか?"],
+    ["Pythonプログラミング言語とは何か簡単に説明してもらえますか?"],
+    ["シンデレラのあらすじを一文で説明してください。"],
+    ["コードを書くときに避けるべきよくある間違いは何ですか?"],
+    ["人工知能と「OpenVINOの利点」について100語程度のブログ記事を書いてください。"],
+]
+
+
+def get_uuid():
+    """
+    universal unique identifier for thread
+    """
+    return str(uuid4())
+
+
+def handle_user_message(message, history):
+    """
+    callback function for updating user messages in interface on submit button click
+
+    Params:
+      message: current message
+      history: conversation history
+    Returns:
+      None
+    """
+    # Append the user's message to the conversation history
+    return "", history + [[message, ""]]
+
+
+def make_demo(run_fn: Callable, stop_fn: Callable, title: str = "OpenVINO Chatbot", language: Literal["English", "Chinese", "Japanese"] = "English"):
```

Comment:
the `stop_fn` will not work in this case, so I think we should set a global object to stop the model request.

Reply:
Gradio demo will be stuck when you start a question after stopping the previous answering